.github/workflows/pr_ci.yml

```diff
@@ -2,6 +2,12 @@ name: PR CI
 on:
   pull_request:
   workflow_dispatch:
+    inputs:
+      runner_os:
+        description: 'The OS of the runners that will build and test bodo. Must be one of {"ubuntu-latest", "windows-latest"}'
+        required: false
+        type: string
+        default: 'ubuntu-latest'
```

Comment:
Optional, but you can define specific options that are allowed. When you trigger the run, there will be a dropdown of all allowed options. Easier than explaining in the description. See https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/.

```
on:
  workflow_dispatch:
    inputs:
      name:
        type: choice
        description: Who to greet
        options: 
        - monalisa
        - cschleiden
```